2-mandatory/4-people-i-know.js

```diff
@@ -378,14 +378,22 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
-
+friends.forEach((friend) => {
```

Comment:
Hi Busra, I wrote my code the way you wrote yours but my tech buddy gave me a better idea of using filter. It actually looks better using filter. 
`let thirtyFiveOrOlder = friends.filter((friend)=> friend.age >= 35);`
All the same well done for your great effort.